tests/modules/keyspace_events.c

```diff
@@ -102,6 +112,51 @@ static int cmdDelKeyCopy(RedisModuleCtx *ctx, RedisModuleString **argv, int argc
     return REDISMODULE_OK;
 }
 
+static int cmdIncrCase1(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
```

Comment:
please add a one liner comment for each of these function to to emphasize their difference.
e.g. "Call INCR and propagate using xxxx"